Makefile

```diff
@@ -303,13 +303,8 @@ image-push: ## Push MULTIARCH_TARGETS images
 .PHONY: manifest-build
 manifest-build: ## Build MULTIARCH_TARGETS manifest
 	@echo 'building manifest $(IMAGE)'
-ifeq (${OCI_BIN}, docker)
+	DOCKER_BUILDKIT=1 $(OCI_BIN) rmi ${IMAGE} -f
 	DOCKER_BUILDKIT=1 $(OCI_BIN) manifest create ${IMAGE} $(foreach target,$(MULTIARCH_TARGETS), --amend ${IMAGE}-$(target));
```

Comment:
this will break on podman since it doesn't have `--amend` but since we default to docker at least out CI should be fine

Reply:
podman does have `--amend` : https://docs.podman.io/en/latest/markdown/podman-manifest-create.1.html#options
It was only the outdated version shipped with the github CI image that didn't have it, so it was only a CI issue, but on our dev machines we are fine with a recent podman. FI I tested on my machine with podman.

Reply:
https://github.com/containers/podman/releases/tag/v4.3.0 or later